channels.go

```diff
@@ -0,0 +1,134 @@
+package telebot
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// ChatInfo struct is the main chat information from telegram
+type ChatInfo struct {
+	ID       int64  `json:"id"`
+	Title    string `json:"title"`
+	Username string `json:"username"`
+	Type     string `json:"type"`
+}
+
+// For each member of a chat , we have some information like usertype (admin , etc..) 
+// and the main information about user like id and etc...
+type ChannelUser struct {
```

Comment:
`ChatMember` is the better name for this structure because it corresponds to the [official documentation](https://core.telegram.org/bots/api#chatmember).